examples/llama7b_w4a16_quantization.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>Maybe important to mention that vLLMQuantizationModifier is in general a PTQ method (for those who are quant-savvy)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/2285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>